src/middlewared/middlewared/api/v25_10_0/core.py

```diff
@@ -4,14 +4,14 @@
 from pydantic import ConfigDict, Field
 
 from middlewared.api.base import (
-    BaseModel, ForUpdateMetaclass, LongString, query_result, single_argument_args, single_argument_result,
+    BaseModel, ForUpdateMetaclass, LongString, single_argument_args, single_argument_result,
 )
-from .common import QueryArgs
 
 __all__ = [
     "CoreGetServicesArgs", "CoreGetServicesResult",
     "CoreGetMethodsArgs", "CoreGetMethodsResult",
-    "CoreGetJobsArgs", "CoreGetJobsResult",
+    "CoreGetJobsItem",
+    "CoreGetJobsAddedEvent", "CoreGetJobsChangedEvent",
```

Comment:
It would be nice if this API included more documentation.

Reply:
About what?

Reply:
I meant the whole module, but meh.  Doesn't have to be done as part of this PR.